src/braket/juliapkg.json

```diff
@@ -3,8 +3,7 @@
     "packages": {
         "BraketSimulator": {
             "uuid": "76d27892-9a0b-406c-98e4-7c178e9b3dff",
-            "url": "https://github.com/amazon-braket/BraketSimulator.jl.git",
-            "rev": "main"
+            "rev": "ksh/nobraket"
```

Comment:
this change would be reverted once https://github.com/amazon-braket/BraketSimulator.jl/pull/36 is merged, correct?

Reply:
Yes. For now I include it so we can test this branch with notebooks and so that CI points to the right julia branch. Once that PR is merged, we can tag a proper Julia version of `BraketSimulator.jl` and point to that version, rather than to the branch head.